vault/resource_ldap_auth_backend_test.go

```diff
@@ -284,7 +285,8 @@ resource "vault_ldap_auth_backend" "test" {
     discoverdn             = false
     deny_null_bind         = true
     description            = "example"
-	userfilter             = "({{.UserAttr}}={{.Username}})"
+    userfilter             = "({{.UserAttr}}={{.Username}})"
```

Comment:
@FalcoSuessgott I allowed myself to not only copy/paste but try to improve the whitespace (spaces) here. Hope that's fine.

Reply:
Good catch! :)